src/pal/src/loader/module.cpp

```diff
@@ -676,7 +676,7 @@ PAL_RegisterModule(
 {
     HINSTANCE hinstance = nullptr;
 
-    int err = PAL_InitializeDLL();
+    int err = PAL_InitializeDLLWithoutSEH();
```

Comment:
This seems pretty hard to reason about. It may be better to just get rid of this whole `PAL_RegisterModule` thing and call `PAL_Initialize` explicitly in each library that has a copy of the PAL statically linked in.